src/shoppingList/repository/shoppingListRepository.js

```diff
@@ -9,4 +9,22 @@ async function createShoppingList(title, userId) {
   });
 }
 
-export { createShoppingList };
+async function findShoppingListById(id) {
+  return await prismaClient.shoppingList.findUnique({
```

Comment:
redundant use of await
```suggestion
  return prismaClient.shoppingList.findUnique({
```

Reply:
pq é redundante ?

Reply:
Pq uma função async sempre retorna uma promise. Vc retornar uma promise ou retornar o valor “awaited” da promise não vai fazer diferença, o return type da função ainda vai ser o mesmo